test/libsolidity/syntaxTests/iceRegressionTests/oversized_var.sol

```diff
@@ -4,11 +4,16 @@ contract b {
     }
 
     c d;
+
     function e() public view {
         c storage x = d;
-	x.a[0];
+        x.a[0];
+        function()[3**44] storage b;
```

Comment:
Maybe rename it from `b` to something else so the first new error (unrelated to this PR) vanishes?